### PR TITLE
Remove a few warnings

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: core/src/main/java/org/mapfish/print/http/HttpRequestCache.java [hunk 1]
```diff
@@ -54,16 +54,10 @@ public CachedClientHttpResponse(final ClientHttpResponse originalResponse) throw
             this.status = originalResponse.getRawStatusCode();
             this.statusText = originalResponse.getStatusText();
             this.cachedFile = File.createTempFile("cacheduri", null, HttpRequestCache.this.temporaryDirectory);
-            InputStream is = originalResponse.getBody();
-            try {
-                OutputStream os = new FileOutputStream(this.cachedFile);
-                try {
+            try (InputStream is = originalResponse.getBody()) {
```
Comment (sbrunner) [inline core/src/main/java/org/mapfish/print/http/HttpRequestCache.java]: It does the same as the with python statement?

File: core/src/main/java/org/mapfish/print/http/MfClientHttpRequestFactoryImpl.java [hunk 2]
```diff
@@ -166,14 +166,9 @@ protected Response executeInternal(@Nonnull final HttpHeaders headers) throws IO
                 }
             }
             if (this.request instanceof HttpEntityEnclosingRequest) {
-                HttpEntityEnclosingRequest entityEnclosingRequest = (HttpEntityEnclosingRequest) this.request;
-                Closer closer = Closer.create();
```
Comment (sbrunner) [inline core/src/main/java/org/mapfish/print/http/MfClientHttpRequestFactoryImpl.java]: What's was that Closer?

File: core/src/main/java/org/mapfish/print/http/HttpRequestCache.java (hunk 1 above)
Comment (pvalsecc) [reply core/src/main/java/org/mapfish/print/http/HttpRequestCache.java]: Yes

File: core/src/main/java/org/mapfish/print/http/MfClientHttpRequestFactoryImpl.java (hunk 2 above)
Comment (pvalsecc) [reply core/src/main/java/org/mapfish/print/http/MfClientHttpRequestFactoryImpl.java]: Nothing was registered, so it was closing nothing.

**Pull request merged**